test/unit/algorithm_toolkit/test_hyperparameter_validation.py

```diff
@@ -187,3 +187,72 @@ def test_simple_tuple(self):
 
         result = hyperparameters.validate({"tuple_": "(1,0,-1)"})
         self.assertEqual(result["tuple_"], (1, 0, -1))
+
+    def test_tuple_alias(self):
+        hyperparameters = hpv.Hyperparameters(hpv.TupleHyperparameter(name="tuple_",
+                                                                           range=[-1, 0, 1],
+                                                                           required=False))
+        hyperparameters._declare_alias("tuple_", "tup_")
+
+        with self.assertRaises(exc.UserError):
+            hyperparameters.validate({"tup_": "(1,0,-1,2)"})
+
+        result = hyperparameters.validate({"tup_": "(1,0,-1)"})
+        self.assertEqual(result["tuple_"], (1, 0, -1))
+
+    def test_simple_alias(self):
+        hps = hpv.Hyperparameters(hpv.ContinuousHyperparameter(name="gamma",
+                                                               range=hpv.Interval(min_closed=0, max_closed=1),
+                                                               required=True))
+        hps._declare_alias("gamma", "min_split_loss")
+
+        with self.assertRaises(exc.UserError):
+            hps.validate({})
+
+        with self.assertRaises(exc.UserError):
+            hps.validate({"min_split_loss": "ha"})
```

Comment:
same comment as above - is this test testing functionality introduced as a part of this commit/code change?